### PR TITLE
Kristin staff

### PR DESCRIPTION
**Pull request by krixstin**
developed the front of **staff** section

- add new staff form
- add existing staff form
- list of currently assigned(?) staffs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,6 +26,7 @@
     "express": "^4.17.1",
     "express-handlebars": "^4.0.3",
     "express-session": "^1.17.2",
+    "expression": "^0.0.1",
```
Comment (cs-paz) [inline package.json]: Do we need this installed?  I don't see it being imported anywhere, if not just make a commit after you `npm un expression`

**Review by cs-paz — CHANGES_REQUESTED** [review]
Overall, this looks good, but we should merge the current master into this and make CSS changes so that this aligns with @ashnarazdan123 's latest changes to CSS.  This branch was made before the main css changes were made

File: package.json (hunk 1 above)
Comment (krixstin) [reply package.json]: Not sure when & why I installed that. I just uninstalled it and committed the change. Thanks!

File: views/cases/staff.hbs [hunk 2]
```diff
@@ -1,3 +1,77 @@
-<div>
-    Staff
-</div>
\ No newline at end of file
+<div class='row justify-content-center' style="margin-top: 10px;">
+    <h3> Staff </h3>
+</div>
+
```
Comment (cs-paz) [inline views/cases/staff.hbs:4]: Let's wrap the form markup underneath this with `<div class="card body p-2" style="border-radius: 1rem; width: 50rem; margin: auto;"></div>` and stop before u get to the table with the test data, and merge conflicts need to be resolved before this PR can be approved.  If you need any help resolving the conflicts, let me know

File: views/cases/staff.hbs [hunk 3]
```diff
@@ -1,3 +1,77 @@
...
+            </select>
+        </div>
+    
+        <div class="col-sm-1">
+            <button type="button" class="btn btn-secondary btn-sm">
+                Add 
+            </button>
+        </div>
+    </div>
+
```
Comment (cs-paz) [inline views/cases/staff.hbs:60]: place ending div here

**Review by cs-paz — CHANGES_REQUESTED** [review]

**Review by cs-paz — APPROVED** [review]
Looks great!

**Review by ezwendel — APPROVED** [review]

**Pull request merged**